mlir/lib/Transforms/InlinerPass.cpp

```diff
@@ -40,11 +40,7 @@ static void defaultInlinerOptPipeline(OpPassManager &pm) {
 namespace {
 class InlinerPass : public impl::InlinerBase<InlinerPass> {
 public:
-  InlinerPass();
-  InlinerPass(const InlinerPass &) = default;
-  InlinerPass(std::function<void(OpPassManager &)> defaultPipeline);
-  InlinerPass(std::function<void(OpPassManager &)> defaultPipeline,
-              llvm::StringMap<OpPassManager> opPipelines);
+  InlinerPass(const InlinerConfig &inlinerConfig, unsigned inliningThreshold_);
```

Comment:
How do we dump the passing configuration? How do I set it from the command line pass-pipeline? 
I would think that would require hooking the InlinerConfig into the cl::opt infrastructure (parsing/printing)